src/components/Main.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { TodoList } from './TodoList';
+
+export const Main: React.FC = () => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings